engines/mohawk/myst_state.h

```diff
@@ -96,6 +96,11 @@ enum DniEnding {
 						   // and destroying the books of his sons
 };
 
+enum SavegameVersions {
```

Comment:
Save version handling is already implemented there: https://github.com/dafioram/scummvm/blob/8bc6e3d3b69a874d88baa5e8dcd83fb2d9e82653/engines/mohawk/myst_state.cpp#L49